go/cache/lru_cache_test.go

```diff
@@ -167,19 +158,15 @@ func TestLRUIsEvicted(t *testing.T) {
 	// lru: [key0, key1, key2]
 
 	// The least recently used one should have been evicted.
-	if _, ok := cache.Get("key3"); ok {
-		t.Error("Least recently used element was not evicted.")
-	}
+	v, ok := cache.Get("key3")
+	assert.False(t, ok, "Least recently used element was not evicted: %v", v)
 
-	if e, want := cache.Evictions(), int64(1); e != want {
-		t.Errorf("evictions: %d, want: %d", e, want)
-	}
+	e := cache.Evictions()
+	assert.EqualValues(t, 1, e, "evictions: got %d, want: %d", e, 1)
```

Comment:
No ned to report expected/found because `assert.EqualValues(...)` already does so.
```suggestion
	assert.EqualValues(t, 1, e)
```